src/quicksort.nr

```diff
@@ -1,2 +1,4 @@
 pub mod quicksort;
 pub mod quicksort_explicit;
+pub mod quicksort_recursive;
+pub mod quicksort_recursive_explicit;
```

Comment:
I don't think that we should preserve the recursive flavour if we have an equivalent non-recursive flavour.